pillar.example

```diff
@@ -105,7 +105,16 @@ nginx:
       - upstream netdata:
           - server: 127.0.0.1:19999
           - keepalive: 64
-
+  
```

Comment:
There are whitespaces after the closing bracket, that tests are showing. Can you remove them?